package ru.sagutdinov.exception

class UserExistsException(message: String): RuntimeException(message)